observation_chronicle/snow/t06z/adpsfc_snow.yaml

```diff
@@ -0,0 +1,76 @@
+# Instrument metadata
+# -------------------
+commissioned: 1970-01-01T00:00:00
+
+observer_type: conventional  # Type of chronicle to use
+
+window_option: max
+
+# observation type initial configuration
+# --------------------------------
+stations_to_reject: [71120, 71397, 71621, 71727, 71816  # from North America
+  , 10863, 16179 # from Europe
+  , 40550, 40580, 40582, 40587, 40592 # Mid East
+  , 47005, 47008, 47014, 47016, 47020, 47022, 47025, 47028, 47031, 47035 # from North Korea
+  , 47037, 47039, 47041, 47046, 47050, 47052, 47055, 47058, 47060, 47061
+  , 47065, 47067, 47068, 47069, 47070, 47069, 47070, 47075, 47090
+  , 48698, 48830 # from Singapore
+  , 50136, 50353, 50468, 50557, 50564, 50603, 50658, 50745, 50756, 50774 # from China
```

Comment:
Can we get add a "justification" line to identify these stations as the ones that have 0 at the 6,12,18 cycle. 

Also, I thought this problem started in 2019, but if I'm interpreting this correctly we're discarding these stations from 1970 onwards.

Reply:
Thanks @ClaraDraper-NOAA for the review. Do you mean to add a comment line showing the reason of removing these stations?

Here, we assumed China provide the bad snow reports at 06/12/18z cycles when they started to provide their snow reports. At such situation, we added these stations in the initial list and it should be the same for the commissioned dates at 1970 or 2019. However, if China started with good snow reports at 06/12/18z cycle then changed to with bad report at 2019, we would have to move these stations out of the initial list.

Reply:
To make sure I understand, these stations were first introduced in 2019?  

Yes - a comment in the "justification" line saying something like ("NE China stations incorrectly set to 0 mm at 6,12,18. UTC"